NitroxClient/MonoBehaviours/Chat.cs

```diff
@@ -20,14 +20,14 @@ public void OnConsoleCommand_chat(NotificationCenter.Notification n)
             {
                 String text = "";
 
-                for(int i = 0; i < n.data.Count; i++)
+                for (int i = 0; i < n.data.Count; i++)
```

Comment:
While you're at it, can you change this to a `string.Join` structure?

Reply:
n is a NotificationCenter.Notification not a string[]. Is there a conversion?

Reply:
`.Select(n => n.ToString())`

Reply:
n.data is a Hashtable. No definition for Select() :(

Reply:
A `Hashtable` implements `IEnumerable`, not `IEnumerable<object>`, so you have to cast it to a generic `IEnumerable` first before methods like `.Select` become available.

```csharp
string text = string.Join(" ", n.data.Cast<object>().Select(d => d.ToString()).ToArray());
```

Reply:
Dosn't work. Just show the Type...

Reply:
Have you tried the "potential fixes" menu (<kbd>ctrl</kbd>+<kbd>.</kbd>)? It should tell you to `using System.Linq;`.

Reply:
Yes I have used ```System.Linq```.